keyboards/geistmaschine/geist/readme.md

```diff
@@ -0,0 +1,25 @@
+# Geistmaschine Geist
+
+![Geist](https://i.imgur.com/qpb6Rh9.png)
+
+Geist is a high-end 65% keyboard, featuring a rotary encoder with a large knob.
+* Keyboard Maintainer: [ebastler](https://github.com/ebastler)
+* Hardware Supported: Geist Multi-Layout and Hot-Swap Rev1 (to access all multi-layout PCB options, use `default` keymap. Hotswap can work with either `default` or `ansi`)
+* Hardware Availability: [geistmaschine.io](https://geistmaschine.io/)
+
+Make example for this keyboard (after setting up your build environment):
+
+    qmk compile geistmaschine/geist -km default
+
+Flashing example for this keyboard:
+
+    qmk flash geistmaschine/geist -km default
```

Comment:
```suggestion
    make geistmaschine/geist:default

Flashing example for this keyboard:

    make geistmaschine/geist:default:flash
```

Reply:
I have already merged multiple boards using the "new" compile/flash syntax in readme, and even specifically asked about which of the two to use a while ago, was told both are acceptable. Do I have to apply this? I'd rather keep it in line with my other recently merged boards.

Reply:
No, that's fine.